CONTRIBUTING.md

```diff
@@ -36,31 +44,8 @@ When you are adding code to Prebid.js, or modifying code that isn't covered by a
 - If you need to check `adloader.loadScript` in a test, use a `stub` rather than a `spy`. `spy`s trigger a network call which can result in a `script error` and cause unrelated unit tests to fail. `stub`s will let you gather information about the `adloader.loadScript` call without affecting external resources
 - When writing tests you may use ES2015 syntax if desired
 
-### Running tests
-After checking out the Prebid.js repository and installing dev dependencies with `npm install`, use the following commands to run tests as you are working on code:
-
-- `gulp test` will run the test suite once (`npm test` is aliased to call `gulp test`)
-- `gulp serve` will run tests once and stay open, re-running tests whenever a file in the `src` or `test` directory is modified
-
-### Checking results and code coverage
-Check the test results using these guidelines:
-
-- Look at the total number of tests run, passed, and failed in the shell window.
-- If all tests are passing, great.
-- Otherwise look for errors printed in the console for a description of the failing test.
-- You may need to iterate on your code or tests until all tests are passing.
-- Make sure existing tests still pass.
-- There is a table below the testing report that shows code coverage percentage, for each file under the `src` directory.
-- Each time you run tests, a code coverage report is generated in `build/coverage/lcov/lcov-report/index.html`.
```

Comment:
Think having a link to this report is good to preserve, maybe just add a heading for `Test Coverage`